avro_schemas/sunav2/sunav2_log_flags.avsc

```diff
@@ -0,0 +1,33 @@
+{
+    "type": "record",
+    "name": "sunav2",
```

Comment:
We don't yet use the "name" field much for science-based avro schemas, but if/when they go in some sort of registry the names need to be unique within the namespace. Please use something like "sunav2_log_flags".